vendor/github.com/Microsoft/go-winio/file.go

```diff
@@ -69,6 +69,7 @@ func initIo() {
 // win32File implements Reader, Writer, and Closer on a Win32 handle without blocking in a syscall.
 // It takes ownership of this handle and will close it if it is garbage collected.
 type win32File struct {
+	sync.Mutex
```

Comment:
I see what you did there, https://github.com/Microsoft/go-winio/pull/61

In the future, can you put a comment in vendor.conf and point it to your fork with the correct hash.